core/auth/v2/authenticator.go

```diff
@@ -67,20 +67,31 @@ func (*authenticator) AuthenticateBlobRequest(header *core.BlobHeader, signature
 // The signature is signed over the byte representation of the account ID and requestHash
 // See implementation of BlobRequestSigner.SignPaymentStateRequest for more details
 func (a *authenticator) AuthenticatePaymentStateRequest(accountAddr common.Address, request *pb.GetPaymentStateRequest) error {
```

Comment:
https://go.dev/wiki/Deprecated, maybe we add in the go doc something like:

Deprecated: Use AuthenticatePaymentStateForAllQuorumsRequest instead.

Reply:
added in [commit](https://github.com/Layr-Labs/eigenda/pull/1613/commits/6fbbfe85965c0eca4eb56febabf58dffe21fd799) and [commit](https://github.com/Layr-Labs/eigenda/pull/1613/commits/936c279d50fa5f71e5cbeb27bd51819a17571236) for GetPaymentState API reques